compiler/jsgen.nim

```diff
@@ -211,6 +211,8 @@ proc mapType(p: PProc; typ: PType): TJSTypeKind =
   if p.target == targetPHP: result = etyObject
   else: result = mapType(typ)
 
+var mangled = initSet[string]()
```

Comment:
I'm sure you find some Context object where this can be put into. Globals make the compiler hard to use as a library.

Reply:
TODO: i think i can remove that